scripts/WinInstallFiles/LaunchJMRI.nsi

```diff
@@ -25,6 +25,10 @@
 ; -------------------------------------------------------------------------
 ; - Version History
 ; -------------------------------------------------------------------------
+; - Version 0.1.29.0
+; - Prepend the "Settings/lib" jars before the prepended "Cp=*", where "Settings"
```

Comment:
Shouldn't this be `append` instead of `prepend`?

`prepend "Cp=*"`

Reply:
Just trying to make it consistent with Linux/OSX as described in [this jmri-developers post](https://jmri-developers.groups.io/g/jmri/message/9421).